source/elements/oneMKL/source/domains/blas/gemm_batch.rst

```diff
@@ -244,7 +245,8 @@ gemm_batch (USM Version)
 
 .. rubric:: Description
 
-The USM version of ``gemm_batch`` supports the group API and strided API. 
+The USM version of ``gemm_batch`` supports the group API and the strided API.
+The group API supports either pointer or span inputs.
```

Comment:
why either/or? It can support both right?

Reply:
Not simultaneously?

Reply:
For instance, for one call all group API parameters need to be either pointer or span inputs.
We can write it with *and* if that is more clear.